optaplanner-core/src/test/java/org/optaplanner/core/api/solver/SolverManagerTest.java

```diff
@@ -66,7 +69,19 @@ public void solveBatch_2InParallel() throws ExecutionException, InterruptedExcep
         assertSolutionInitialized(solverJob2.getFinalBestSolution());
     }
 
-    @Test(timeout = 600_000)
+    private CustomPhaseConfig createPhaseWithConcurrentSolvingStart(int barrierPartiesCount) {
```

Comment:
Is this method reused anywhere else? Otherwise, same arguments against extraction as below

Reply:
It is used by two different tests, I would rather keep it.